### PR TITLE
fix: gcc8.3.0 and memset overflow

### PR DESCRIPTION
**Pull request by fingera**

### CODE REVIEW

**Comment by stgraber** [comment]
You're sending this against the LXC 2.1 branch, LXC 2.1 is unsupported, please retarget this to the master branch if applicable.

File: src/lxc/network.c [hunk 1]
```diff
@@ -2151,8 +2151,8 @@ static int lxc_create_network_unpriv_exec(const char *lxcpath, const char *lxcna
 		return -1;
 	}
 
-	memset(netdev->name, 0, IFNAMSIZ + 1);
```
Comment (fingera) [inline src/lxc/network.c:2154]: char name[IFNAMSIZ];

**Comment by fingera** [comment]
> You're sending this against the LXC 2.1 branch, LXC 2.1 is unsupported, please retarget this to the master branch if applicable.

I plan to use version 2.1
Isn't that a good idea?
Should I use 4.0?

**Comment by fingera** [comment]
4.0 is okay

**Comment by stgraber** [comment]
It's not a good idea to use a release which we don't support anymore, no :)
Non LTS releases like 2.1 (as opposed to 2.0) are only supported until the next release comes out, so that particular release was only supported for a few months.

LXC 2.0, 3.0 and 4.0 are all LTS releases that are still supported, though both 2.0 and 3.0 are now in the security-only part of their life, 4.0 is the latest LTS release with full bugfix and security support.

File: src/lxc/network.c (hunk 1 above)
Comment (fingera) [reply src/lxc/network.c:2154]: @stgraber There seems to be a small security problem here. on 2.x

**Pull request closed without merging**